src/OrchardCore.Modules/OrchardCore.Forms/Migrations.cs

```diff
@@ -53,6 +53,12 @@ public int Create()
                 .WithPart("LabelPart")
                 .Stereotype("Widget"));
 
+
```

Comment:
Add these to an `UpdateFrom3()` method too so existing installations will get them as well.